zerver/views/report.py

```diff
@@ -61,6 +67,7 @@ def json_report_narrow_time(request, user_profile,
 def json_report_unnarrow_time(request, user_profile,
                             initial_core=REQ(converter=to_non_negative_int),
                             initial_free=REQ(converter=to_non_negative_int)):
+    # type: (Any, UserProfile, REQ, REQ) -> HttpResponse
```

Comment:
FYI, the "request" objects should be of type HttpRequest.  The problem is that we do scribble various extra values onto those objects. which may make it hard to annotate them properly.  It's possible we need to make our request objects an explicit subclass of Django's HttpRequest (e.g. ZulipHttpRequest) that adds the extra attributes "officially".